.github/workflows/reusable-performance.yml

```diff
@@ -16,6 +16,11 @@ on:
         required: false
         type: 'string'
         default: '6.1.1'
+      php-version:
+        description: 'The PHP version to use.'
+        required: false
+        type: 'string'
+        default: '8.3'
```

Comment:
Last I checked, many of our workflows rely on the `LOCAL_PHP` value set in the `.env` file of this repo that defaults to `latest`, which currently results in PHP 8.2. [At least some of our reusable workflows](https://github.com/WordPress/wordpress-develop/blob/trunk/.github/workflows/reusable-coding-standards-php.yml#L9-L13) also use `latest` as a default. I think it would be good to make these consistent if possible.

For the Performance workflow, [we need to intentionally pin it to PHP 8.2](https://core.trac.wordpress.org/ticket/60127#comment:23) given that our baseline still uses WP 6.1.1, which doesn't support PHP 8.3.

Reply:
I've gone and updated this because it makes sense for consistency's sake. But wanted to document some findings here because it may actually make sense to pin `8.2` here instead (or whatever the highest version of PHP supported for the baseline version is).

I initially thought that the only reason the workflow has not broken in `trunk` was because `latest` is the default for `LOCAL_PHP` (the `latest` tag is [currently applied to the PHP 8.2 container](https://github.com/WordPress/wordpress-develop/actions/runs/9304967679/job/25610808836#step:11:20)).

However, after pinning `8.3` in the 6.5 branch, the [workflow continued to pass](https://github.com/WordPress/wordpress-develop/actions/runs/9305868021/job/25613806478#step:12:25). @joemcgill do you have any idea why the workflow would fail using PHP 8.3 in the 6.4 branch, but not the 6.5 one?

Reply:
The reason those still pass is because the baseline measurements are only run on push actions to `trunk` (i.e. commits) and not on PRs, so the error doesn't occur.